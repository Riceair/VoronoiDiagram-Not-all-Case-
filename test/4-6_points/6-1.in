6
12 89
124 592
131 11
543 212
23 400
312 12