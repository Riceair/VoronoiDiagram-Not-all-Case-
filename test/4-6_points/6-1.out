P 12 89
P 23 400
P 124 592
P 131 11
P 312 12
P 543 212
E -195339.9 7154.25 194.77 238.23
E -120369.9 63854.25 307.23 373.05
E -49068.5 -74920 194.77 238.23
E 194.77 238.23 220.21 245.29
E 220.21 245.29 276.08 286.9
E 220.21 245.29 849.62 -113677.9
E 276.08 286.9 307.23 373.05
E 276.08 286.9 126219.6 -145177.9
E 307.23 373.05 239733.5 264372
